rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -2911,3 +2875,39 @@ std::string game_list_frame::GetGameVersion(const game_info& game)
 
 	return game->info.app_ver;
 }
+
+void game_list_frame::StopSizeWatcher(bool cancel)
+{
+	if (m_size_watcher.isRunning())
+	{
+		if (cancel)
+		{
+			m_size_watcher.cancel();
+		}
+		m_size_watcher.waitForFinished();
+	}
+}
+
+void game_list_frame::StopRefreshWatcher(bool cancel)
+{
+	if (m_refresh_watcher.isRunning())
+	{
+		if (cancel)
+		{
+			m_refresh_watcher.cancel();
+		}
+		m_refresh_watcher.waitForFinished();
+	}
+}
+
+void game_list_frame::StopRepaintWatcher(bool cancel)
+{
+	if (m_repaint_watcher.isRunning())
```

Comment:
Can simply pass a reference to a single function instead of all these duplicates.

Reply:
That would be possible, but it's not particularly readable

Reply:
I'll do it anyway